app/models.py

```diff
@@ -20,5 +22,18 @@ class Facetname(str, Enum):
     nucleotide = "nucleotide"
     allergen = "allergen"
     trace = "trace"
-    
 
+    @staticmethod
+    def list():
+        return [c.value for c in FacetName]
+
+
+class HungerGameFilter(str, Enum):
+    label = "label"
+    category = "category"
+    brand = "brand"
+    product_weight = "product_weight"
```

Comment:
you should add country, because this is a valid facet (it will superseed country parameter)